python/pydantic_core/_pydantic_core.pyi

```diff
@@ -127,7 +251,32 @@ class SchemaSerializer:
         round_trip: bool = False,
         warnings: bool = True,
         fallback: Callable[[Any], Any] | None = None,
-    ) -> Any: ...
+    ) -> Any:
+        """
+        Serialize a Python object to a Python object including transforming and filtering data.
+
+        Arguments:
+            value: The Python object to serialize.
+            mode: The serialization mode to use, either `'python'` or `'json'`, defaults to `'python'`. In JSON mode,
+                all values are converted to JSON compatible types, e.g. `None`, `int`, `float`, `str`, `list`, `dict`.
+            include: A set of fields to include, if `None` all fields are included.
+            exclude: A set of fields to exclude, if `None` no fields are excluded.
```

Comment:
The types of `include` and `exclude` are `_IncEx` - we may want to make that alias public and document it.

![image](https://github.com/pydantic/pydantic-core/assets/1939362/181f27b9-e44b-4a2e-adae-7bb456a2637f)